sky/clouds/service_catalog/kubernetes_catalog.py

```diff
@@ -26,3 +29,65 @@ def get_image_id_from_tag(tag: str, region: Optional[str]) -> Optional[str]:
 def is_image_tag_valid(tag: str, region: Optional[str]) -> bool:
     """Returns whether the image tag is valid."""
     return common.is_image_tag_valid_impl(_image_df, tag, region)
+
+
+def list_accelerators(
+        gpus_only: bool,
+        name_filter: Optional[str],
+        region_filter: Optional[str],
+        quantity_filter: Optional[int],
+        case_sensitive: bool = True
+) -> Dict[str, List[common.InstanceTypeInfo]]:
+    has_gpu = kubernetes_utils.detect_gpu_resource()
+    if not has_gpu:
+        return {}
+
+    label_formatter, _ = kubernetes_utils.detect_gpu_label_formatter()
+    if not label_formatter:
+        return {}
+
+    accelerators: Set[Tuple[str, int]] = set()
+    key = label_formatter.get_label_key()
+    nodes = kubernetes_utils.get_kubernetes_nodes()
+    for node in nodes:
+        if key in node.metadata.labels:
+            accelerator_name = label_formatter.get_accelerator_from_label_value(
+                node.metadata.labels.get(key))
+            accelerator_count = int(
```

Comment:
It would be nice to list Kubernetes as an option if a specific GPU is passed as an arg to `sky show-gpus`. For example:
```
(base) ➜  ~ sky show-gpus T4:4
*NOTE*: for most GCP accelerators, INSTANCE_TYPE == (attachable) means the host VM's cost is not included.

GPU  QTY  CLOUD  INSTANCE_TYPE          DEVICE_MEM  vCPUs  HOST_MEM  HOURLY_PRICE  HOURLY_SPOT_PRICE  REGION
T4   4    AWS    g4dn.12xlarge          16GB        48     192GB     $ 3.912       $ 1.245            ap-northeast-1
T4   4    Azure  Standard_NC64as_T4_v3  -           64     440GB     $ 4.352       $ 0.435            centralus
T4   4    GCP    (attachable)           -           -      -         $ 1.400       $ 0.279            asia-east1
# Should show Kubernetes too
```

Reply:
Sounds good, I will work on this.

Reply:
I added it like
```python
if method_name == "list_accelerators":
            clouds.append("kubernetes")
```
in _map_clouds_catalog as the Kubernetes catalog does not implement the other catalog methods as of now.